src/sectionproperties/analysis/solver.py

```diff
@@ -118,9 +118,10 @@ def solve_direct_lagrange(
     u = spsolve(A=k_lg, b=np.append(f, 0))
 
     # compute error
-    err = u[-1] / max(np.absolute(u))
+    multiplier = abs(u[-1])
```

Comment:
Need to test both absolute and relative error. If the RHS is trivial, the solution will be close to epsilon, making the relative error randomly large. This causes random failure of tests on different machines.